src/main/asciidoc/transactions.adoc

```diff
@@ -33,82 +41,257 @@ static class Config extends AbstractCouchbaseConfiguration {
-utilize it:
+CouchbaseTransactionalOperator is used for reactive transactions. It is an important class for Couchbase Transactions in Spring Data as there is no Reactive TransactionManager for Couchbase.
+The class simplifies programmatic transaction demarcation and transaction exception handling.
+The central method is transactional(), supporting transactional wrapping of functional sequences code that. This operator handles the transaction lifecycle and possible exceptions such that neither the ReactiveTransactionCallback implementation nor the calling code needs to explicitly handle transactions.
+Typical usage: Allows for writing low-level data access objects that use resources such as database connections but are not transaction-aware themselves. Instead, they can implicitly participate in transactions handled by higher-level application services utilizing this class, making calls to the low-level services via an inner-class callback object.
 
-.Transaction Conversion on Write
+Note that a CouchbaseTransactionalOperator is available as a bean.
+If creating one explicitly, it must be created with the following (and not TransactionalOperator.create(manager)).
+.TransactionalOperator Construction
 ====
 [source,java]
 ----
-@Autowired
-MappingCouchbaseConverter mappingCouchbaseConverter;
+public static CouchbaseTransactionalOperator create(CouchbaseCallbackTransactionManager manager) 
+====
 
-public void doSomething() {
-  transactions.run(ctx -> {
+.Transaction Access Using TransactionalOperator.execute()
+====
+[source,java]
+----
+@Autowired TransactionalOperator txOperator;
 
-   Airline airline = new Airline("demo-airline", "at");
-   CouchbaseDocument target = new CouchbaseDocument();
-   mappingCouchbaseConverter.write(airline, target);
+    Person person = rxCBTmpl.insertById(Person.class).inCollection(cName).one(WalterWhite).block();
+    Flux<Person> result = txOperator.execute((ctx) -> rxCBTmpl.findById(Person.class).one(person.id())
+        .flatMap(p -> rxCBTmpl.replaceById(Person.class).one(p.withFirstName("Walt"))));
+    result.blockLast();
+----
+====
 
-   ctx.insert(couchbaseClientFactory.getDefaultCollection(), target.getId(), target.getContent());
+.Transaction Access Using transactional()
+====
+[source,java]
+----
+@Autowired TransactionalOperator txOperator;
 
-   ctx.commit();
-  });
-}
+    Person person = rxCBTmpl.insertById(Person.class).inCollection(cName).one(WalterWhite).block();
+    Mono<Person> result = rxCBTmpl.findById(Person.class).one(person.id())
+        .flatMap(p -> rxCBTmpl.replaceById(Person.class).one(p.withFirstName("Walt")))
+        .as(txOperator::transactional);
+    result.block();
 ----
 ====
+.Transaction Access  simulating @Transactional
+====
+[source,java]
+----
+@Service
+@Component
+class PersonService {
+
+    final ReactiveCouchbaseOperations personOperationsRx;
```

Comment:
btw one (maybe subjective) recommendation would be to use Reactive instead of Rx, it could be confused with RxJava and is certainly lingo that some users might not understand

Reply:
That's a really good point.